src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml

```diff
@@ -9,206 +9,196 @@
-		   TargetType="utu:TabBar">
-        <Setter Property="Background"
+	<mobile:Style x:Key="CupertinoSlidingSegmentedStyle"
+				  BasedOn="{StaticResource DefaultTabBarStyle}"
+				  TargetType="utu:TabBar">
+		<Setter Property="Background"
 				Value="{ThemeResource CupertinoTertiarySystemFillBrush}" />
-        <Setter Property="IsTabStop"
+		<Setter Property="IsTabStop"
 				Value="False" />
-        <Setter Property="Height"
+		<Setter Property="Height"
 				Value="{StaticResource CupertinoSlidingSegmentedControlHeight}" />
-        <Setter Property="ItemsPanel">
-            <Setter.Value>
-                <ItemsPanelTemplate>
-                    <utu:TabBarListPanel />
-                </ItemsPanelTemplate>
-            </Setter.Value>
-        </Setter>
-        <Setter Property="ItemContainerStyle"
+		<Setter Property="SelectionIndicatorTransitionMode"
+				Value="Slide" />
+		<Setter Property="SelectionIndicatorPlacement"
+				Value="Below" />
+		<Setter Property="ItemsPanel">
+			<Setter.Value>
+				<ItemsPanelTemplate>
+					<utu:TabBarListPanel />
+				</ItemsPanelTemplate>
+			</Setter.Value>
+		</Setter>
+		<Setter Property="ItemContainerStyle"
 				Value="{StaticResource CupertinoSlidingSegmentedItemStyle}" />
-        <Setter Property="Template">
-            <Setter.Value>
-                <ControlTemplate TargetType="utu:TabBar">
-                    <Grid x:Name="TabBarGrid"
-						  CornerRadius="8"
-						  Background="{TemplateBinding Background}"
-						  BorderBrush="{TemplateBinding BorderBrush}"
-						  BorderThickness="{TemplateBinding BorderThickness}"
-						  Height="{TemplateBinding Height}">
-                        <utu:TabBarSelectionIndicatorPresenter Owner="{Binding RelativeSource={RelativeSource TemplatedParent}}"
-															   x:Name="SelectionIndicatorPresenter"
-															   AutomationProperties.AutomationId="SelectionIndicatorPresenter"
-															   IndicatorTransitionMode="Slide"
-															   Opacity="0">
-                            <toolkit:ElevatedView Width="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=TemplateSettings.SelectionIndicatorWidth}"
-												  Height="28"
-												  Background="Transparent"
-												  CornerRadius="8"
-												  Elevation="5"
-												  ShadowColor="#CC94949A">
-                                <Rectangle Fill="{ThemeResource SystemBackgroundColor}"
-										   VerticalAlignment="Stretch"
-										   HorizontalAlignment="Stretch"
-										   RadiusX="8"
-										   RadiusY="8"
-										   Height="24"
-										   Width="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}, Path=TemplateSettings.SelectionIndicatorWidth, Converter={StaticResource DeflateWidthConverter}}" />
-                            </toolkit:ElevatedView>
-                        </utu:TabBarSelectionIndicatorPresenter>
-
-                        <ItemsPresenter Padding="{TemplateBinding Padding}" />
```

Comment:
You mean keep the <Setter Property="Template"/> right? When I leave it in, the style seems to ignore the other <Setter Property="SelectionIndicatorPresenterContentTemplate"/> completely because the indicator isn't visible, I committed it to the branch though

Reply:
So I think I had to add a ContentPresenter here instead of the SelectionIndicatorPresenter: https://github.com/unoplatform/uno.toolkit.ui/blob/145a6041330ff2b9c97e54365edc35ee9c07c655/src/library/Uno.Toolkit.Cupertino/Styles/Controls/SlidingSegmentedControl.xaml#L419

`<ContentPresenter ContentTemplate="{TemplateBinding SelectionIndicatorContentTemplate}" />`

But now the indicator takes up the whole width and doesn't move when I change tabbar selection (even if I set the width to "100" the position of the indicator doesn't change)

Reply:
Try adding the Template Setter as the last Setter in the Style, ordering matters on Uno for some reason when it comes to the Template

Reply:
Yeah it's the same behavior whether I put the Template last or not, I'll push that change though

Reply:
Hmm, ok I will take a look

Reply:
Wait... I think I messed up, you don't need to redefine the Template. Just the SelectionIndicatorContentTemplate

Reply:
Did it fix the issues?

Reply:
Yeah! Working great now